nsq_metrics_to_kairos/nsq_metrics_to_kairos.go

```diff
@@ -68,23 +71,25 @@ func main() {
 	if _, err := os.Stat(*confFile); err == nil {
 		conf, err := globalconf.NewWithOptions(&globalconf.Options{Filename: *confFile})
 		if err != nil {
-			log.Fatal(err)
+			stdlog.Fatal(err)
 			os.Exit(1)
 		}
 		conf.ParseAll()
 	}
 
+	log.NewLogger(0, "console", fmt.Sprintf(`{"level": %d, "formatting":true}`, *logLevel))
+
 	if *showVersion {
 		fmt.Println("nsq_metrics_to_kairos")
 		return
 	}
 	hostname, err := os.Hostname()
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(0, err.Error())
```

Comment:
the first value for log.Fatal is the "skip" number (number of entries in the stack to backtrack when printing a stack trace.)
To have the stack trace start from where here, set the skip to 4. Otherwise the stack strace will start from within the log handler.